test/parallel/test-stream-readable-manualStart.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+
+const { Readable } = require('stream');
+
+{
+  let didRead = true;
```

Comment:
Shouldn't this be `false`? Otherwise the assertion isn't really checking anything.